package.json

```diff
@@ -99,13 +104,15 @@
   "devDependencies": {
     "husky": "^3.0.5",
     "lint-staged": "^9.2.5",
-    "prettier": "^1.18.2"
+    "prettier": "^1.18.2",
+    "typescript": "^3.6.3"
   },
   "scripts": {
     "lint": "eslint $(find cli bin -type f) --quiet --cache",
     "pretty": "prettier --write  cli bin",
     "preversion": "git update-index --no-assume-unchanged package-lock.json",
-    "postversion": "git update-index --assume-unchanged package-lock.json && git push --tags && git push"
+    "postversion": "git update-index --assume-unchanged package-lock.json && git push --tags && git push",
+    "check-types": "tsc"
```

Comment:
should probably use `--noEmit` flag, we don't actually want typedef files. Also, probably useless in this project, but the idea is to have type checking before commit.

Reply:
proton-pack won't be in TS ;)